man/man1/mash-job-add.1

```diff
@@ -1,8 +1,92 @@
+.\" Process this file with
+.\" groff -man -Tascii mash-job-add.1
+.\"
 .TH "MASH JOB ADD" "1" "27-Aug-2018" "" "mash job add Manual"
 .SH NAME
-mash\-job\-add \- Send add job request to mash server based on...
+mash\-job\-add \- Add a job to the mash server pipeline
 .SH SYNOPSIS
 .B mash job add
-[OPTIONS] DOCUMENT
+[OPTIONS]
 .SH DESCRIPTION
-Send add job request to mash server based on provided json document.
\ No newline at end of file
+Create a job on the \fImash\fP server for execution in the pipeline.
+.SH OPTIONS
+.TP
+\fB--account\fP
```

Comment:
There is the option to pass in a list of groups and a list of accounts. With the accounts option the parameters are a dictionary and in each framework that dictionary is different.

For example you can pass in a list of ec2 accounts and for each account you can have a list of regions.

The complexity of options is not accounted for here with `--account` and `--regions`.

Reply:
The complexity is hidden in the account setup. Note how this says "--account" and not "--accounts" the user gets to specify only one name. Then "--regions" is expected to match the regions that match the partitions in this account.